### ISSUE TITLE
Color Codes in Write to String

### ISSUE THREAD

**Issue by felipefoz** [JamesMc86/G-CLI#146]
Hi,

Maybe silly question. Is it possible to colors on outputs (ascii colors codes).
Initial attempts seems to not work.
In bash is usually set as:
`$ echo -e "\e[31mRed Text\e[0m"`

**Comment by JamesMc86** [comment]
Not a silly question at all but I don't have a good answer!

Firstly they would need to go in the correct format in the strings in LabVIEW. I'm not sure if that would necessarily be the same way you do it in the terminal. FOr example in the explanation above I think the \e is actually an ESC character. You would need to make sure that is what you are generating in LabVIEW.

There is some conversion that takes place as LabVIEW uses ASCII and rust uses UTF but I would expect these escape characters to survive that. However perhaps that is interfering.

So in short I think it should work but never tried it and might need some experimentation!

**Comment by felipefoz** [comment]
Hypothetically, could be this a feature of G-CLI?

**Comment by felipefoz** [comment]
Solution is in the reply above: https://github.com/JamesMc86/G-CLI/issues/146#issuecomment-1573869779

**Comment by felipefoz** [comment]
In case anyone is looking for something similar. I published a package at:

https://www.vipm.io/package/felipefoz_lib_lv_rainbow_terminal/
